Biahun_Uladzimir/02.Git.local/02.Git.local.md

```diff
@@ -0,0 +1,74 @@
+ 1003  git init
+ 1004  git config --global user.name "Biahun Uladzimir"
+ 1005  git config --global user.email "begun74@gmail.com"
+ 1006  git config user.name "Biahun Uladzimir"
+ 1007  date >> master.txt
+ 1008  git add master.txt
+ 1009  git commit -a -m "Init master"
+ 1010   date >> master.txt
+ 1011  git commit -a -m " === 2 commit ==="
+ 1012  git log --oneline --decorate --graph --all
```

Comment:
need output

Reply:
*   c84771d (HEAD, master) Merge branch 'dev'
|\
| * 5c4d5b7 (dev)  === 10 commit ===
| * e9b1746  === 9 commit ===
| * b5ff317  === 6 commit ===
| * 70f4cac  === 5 commit ===
* |   6631988 Merge branch 'hotfix/we_gonna_die'
|\ \
| * | 7b96b3c  === 13 commit ===
* | | 214eb0c  === 12 commit ===
* | | 5746a77  === 11 commit ===
|/ /
* | 15f23c8  === 8 commit ===
* | 080c739  === 7 commit ===
* | 8db6937  === 4 commit ===
* | fb87e98  === 3 commit ===
|/
* 199e494  === 2 commit ===
* 9477ce6 Init master

Reply:
git log --oneline
c84771d Merge branch 'dev'
6631988 Merge branch 'hotfix/we_gonna_die'
7b96b3c  === 13 commit ===
214eb0c  === 12 commit ===
5746a77  === 11 commit ===
5c4d5b7  === 10 commit ===
e9b1746  === 9 commit ===
15f23c8  === 8 commit ===
080c739  === 7 commit ===
b5ff317  === 6 commit ===
70f4cac  === 5 commit ===
8db6937  === 4 commit ===
fb87e98  === 3 commit ===
199e494  === 2 commit ===
9477ce6 Init master

Reply:
Excuse me May I see example markup? I don't understand

Reply:
> Excuse me May I see example markup? I don't understand
https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet

Reply:
Ok. Where his use  for example?